src/Shared/Microsoft.AspNetCore.Razor.Serialization.Json/ObjectReaders.cs

```diff
@@ -341,7 +341,7 @@ public static RazorProjectInfo ReadProjectInfoFromProperties(JsonDataReader read
             throw new RazorProjectInfoSerializationException(SR.Unsupported_razor_project_info_version_encountered);
         }
 
-        var serializedFilePath = reader.ReadNonNullString(nameof(RazorProjectInfo.SerializedFilePath));
+        var serializedFilePath = reader.ReadString(nameof(RazorProjectInfo.SerializedFilePath));
```

Comment:
Surprised to not see a change to the writer. Does it's absence mean we don't need a version change?

Reply:
A change isn't needed in the writer because the `JsonDataWriter.Write(...)` string overloads already allow nullable strings:

https://github.com/dotnet/razor/blob/2f71d656ec23e4649ee80d2433874cc7bc896df4/src/Shared/Microsoft.AspNetCore.Razor.Serialization.Json/JsonDataWriter.cs#L119-L128